src/components/procedure/_ignore-individual-finding-no-grouping.mdx

```diff
@@ -0,0 +1,7 @@
+1. On the Semgrep Code [Findings](https://semgrep.dev/orgs/-/findings?tab=open) page, click the **Status** filter, and then select **Open** status to see all open findings.
+1. Next to a finding you want to ignore, click the **Open**.
```

Comment:
```suggestion
1. Next to a finding you want to ignore, click **Open**.
```
?

Reply:
@pabloest true, thanks for the suggestion (it is also part of our style guide).